gulpfile.js

```diff
@@ -60,16 +51,41 @@ gulp.task('default', ['clean'], function() {
       path.basename = file_name
       path.extname = ".html"
     }))
+    .pipe(html_replace({
+      title: {
+        src: String(file_name),
+        tpl: '<title>%s</title>'
+      }
+    }))
     .pipe(dest)
     .pipe(open('<%=file.path%>', {app: 'firefox'}))
-    //.pipe(debug())
+    .pipe(debug())
 
 })
 
+gulp.task('get-filename',['clean'], function(cb) {
+  if(content == undefined) {
+    getContent()
+  }
+  content.pipe(through.obj(function(file, enc, callback) {
+    first_non_empty_line = getFirstNonEmptyLine(String(file.contents).split("\n"))
+    file_name = slug(first_non_empty_line, "_")
+    console.log("Found title: " + file_name)
+    callback()
+  }))
+  cb()
```

Comment:
Is there any use case we might need this callback `cb` ? If not, can we remove it to avoid confusion?